File: python_project/Strings/DistanceCaliculator.py
s1 = "abc"
s2 = "xyz"
totalDistance = 0

for i in range(0, len(s1)):
    
    val1 = ord(s1[i])
    val2 = ord(s2[i])
    
    ascii1 = val1
    ascii2 = val2
    
    print("The ascii value of " + str(val1) + " " + str(ascii1))
    print("The ascii value of " + str(val2) + " " + str(ascii2))
    
   
    currentdistance = abs(val1 - val2)
    print(str(currentdistance) + "========")
    
    totalDistance = totalDistance + currentdistance

print("String 1: " + s1)
print("String 2: " + s2)
print("---------------------------------")
print("Cumulative ASCII Distance: " + str(totalDistance))

File: java_project/String/DistanceCaliculator.java
package String;

public class DistanceCaliculator {
    public static void main(String[] args) {
        String s1 = "abc";
        String s2 = "xyz";
        int totalDistance =0;
        for (int i =0; i<s1.length(); i++){
            int val1 = s1.charAt(i);
            int val2 = s2.charAt(i);
            
            int ascii1= (int)val1;
            int ascii2 = (int)val2;

            
            System.out.println("The ascii value of "+val1+" "+ ascii1);
            System.out.println("The ascii value of "+val2+" "+ ascii2);

            int currentdistance = Math.abs(val1-val2);
            System.out.println(currentdistance+"========");
            totalDistance= totalDistance+ currentdistance;
        }
        System.out.println("String 1: " + s1);
        System.out.println("String 2: " + s2);
        System.out.println("---------------------------------");
        
        System.out.println("Cumulative ASCII Distance: " + totalDistance);
        

    }
}
